### DISCUSSION TITLE
Restrict AWS SG from helm chart

### DISCUSSION THREAD

**Discussion by Luisbitvavo** [gravitational/teleport#11871] [Q&A]
Hello all. 
We noticed by deploying teleport-cluster via helm chart using Service: LoadBalancer, it creates Inbound rules in EKS SG with 0.0.0.0/0. We would like to restrict this. Is there any possibility for this?
Thank you.

**Comment by webvictim** [comment]
I don't know this for sure, but I would presume that the [service.beta.kubernetes.io/load-balancer-source-ranges](https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.2/guide/service/annotations/#lb-source-ranges) annotation would change the SG rules. Let me know how you get on.
